### ISSUE TITLE
Error installation

### ISSUE THREAD

**Issue by operatorone** [webrealisierung-ch/onepage-bundle#1]
Hi,

after pasting "composer require wr/onepage-bundle" in the shell, unfortunately I get the following message.

```
[InvalidArgumentException]                                                                                                                         
  Could not find package wr/onepage-bundle at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability 
```

I would love to use your extension. Can I install it manually also.

Tilman

**Comment by danielsteuri** [comment]
Hi Tilman

Thank you for your request!

Please note that this bundle is not stable yet. But of course feel free to use it anyway.

To install the bundle with composer you can add the ```dev-master``` tag in the ```composer require``` statement.

The following statement should work:
```composer require wr/onepage-bundle:dev-master```

As soon we have released the stable version 1.0 you can use the statement without the ```dev-master``` tag.

Let us know if this solves the problem.

Thanks!

Daniel

**Comment by danielsteuri** [comment]
The version 1.0.0 is released. ```dev-master``` is no longer mandatory.
